canonical/canonical.com
# Dev created /solutions/ai/genai

## Done

Dev created /solutions/ai/genai

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check for all things related to addition of new page.
- [Design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=873-6710&node-type=instance&m=dev)
- [Copydoc](https://docs.google.com/document/d/1eMvLlW6U-3ZWRRtY3Bq2u6EaFRqop4n6x8wgj3OWw-k/edit?tab=t.0)
- [Demo](https://canonical-com-1402.demos.haus/solutions/ai/genai)

## Issue / Card

Fixes # [WD-15327](https://warthogs.atlassian.net/browse/WD-15327)

## Screenshots

[if relevant, include a screenshot]


[WD-15327]: https://warthogs.atlassian.net/browse/WD-15327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
